frontend_tests/test_generator.py

```diff
@@ -3,7 +3,7 @@
 import nose2
 import time
 import os
-from happy_scenarios import scenario_full, scenario_short, scenario_x
+from happy_scenarios import *
```

Comment:
`import *` makes my python heart unhappy. You could `import happy_scenarios as hs` if you don't want to import things explicitly.